docker/.env

```
@@ -242,77 +242,41 @@ REGISTER_ENABLED=1
-# - `SANDBOX_LOCAL_*`
-# - `SANDBOX_E2B_*`
-# - `SANDBOX_ALIYUN_CODEINTERPRETER_*`
+# Sandbox provider type and runtime settings are configured in Admin > Sandbox
+# Settings.
 
 # Enable sandbox support.
-# SANDBOX_ENABLED=1
-# COMPOSE_PROFILES=${COMPOSE_PROFILES},sandbox
-# SANDBOX_PROVIDER_TYPE=${SANDBOX_PROVIDER_TYPE:-self_managed}
+SANDBOX_ENABLED=1
+COMPOSE_PROFILES=${COMPOSE_PROFILES},sandbox
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The COMPOSE_PROFILES key is duplicated

(DuplicatedKey)